lib/engine/game/base.rb

```diff
@@ -2760,6 +2760,18 @@ def train_limit_header
       def train_power?
         false
       end
+
+      # This is a bit of a hack to get around how 1840 and 18USA have different and incompatible opinions on what the
+      #  option to use the scrap_trains action in a merger round means
+      def use_1840_style_merger_round_scrap_trains?
```

Comment:
uhhhhh can we fix the hack?

Reply:
It's definitely a hack that probably shouldn't have been in the code base. I'd prefer to write an issue to have it resolved as neither of us know enough about 1840 to make the change. Maybe @Zwergenpunk has thoughts?

Reply:
oh crap :D
in 1840 the merger round is the "buy tram lines". There is a "real" buy and a scrap train step in it.

I am not sure how to resolve that other than chris suggested. I need a real view and chris does not need a real view, just the action.
Maybe the name could be clearer, or we could add a "show_action_view?" to the step. This would put the logic more into the steps and away from the game.

Did that somehow help? ;)

Reply:
Let's rename to what the method actually does "train_actions_always_use_operating_round_view".